File: socket/src/main/java/com/paymong/battle/battle/dto/response/BattleMessageResDto.java
package com.paymong.battle.battle.dto.response;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class BattleMessageResDto {
    private Integer nowTurn;
    private String nextAttacker;
    private Double healthA;
    private Double healthB;
    private Double damageA;
    private Double damageB;
}

File: android/data/src/main/java/com/paymong/data/watch/dto/response/BattleMessageResDto.kt
package com.paymong.data.watch.dto.response

data class BattleMessageResDto(
    val battleRoomId: String,
    val nowTurn: Int,
    val nextAttacker: String,
    val order: String,
    val damageA: Double,
    val damageB: Double,
    val healthA: Double,
    val healthB: Double
)
